server/views/routes/emails.js

```diff
@@ -0,0 +1,21 @@
+const router = require('express').Router();
+const User = require('../../models/user');
+const responseHandler = require('../../utils/response-handler');
+
+// Get check if email is alredy created
+router.get('/:email', async (req, res) => {
+  try {
+    const { email } = req.params;
+    const userEmail = await User.getEmail(email);
```

Comment:
Following the MVC pattern, calls to your Model layer should be abstracted away from your View using a Controller.
M - model => where you define the schema for your data as well as static methods to access or manipulate the data
V - view => (express endpoints) the entry point of your API which handles input from your user and returns output and status
C - controller => all your business logic related to your data model

You'll find it quite useful to break it up like this when you business logic starts to expand.

You can refactor this is in another PR. The blog backend is a decent example of this pattern and how to structure these files (https://bitbucket.t4g.com/projects/MSD/repos/demo-blog---backend/browse/src)